sourcegraph/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "integration_id": "sourcegraph-datadog-services-map-extension",
```

Comment:
This is a bit long for an integration id. Would it be possible to change this to `sourcegraph` or something shorter? Keep in mind, this would be the unique identifying name for the integration.

Reply:
Got it! There may be other types of Sourcegraph integrations in the future so how about `sourcegraph-services-map`? Feel free to make that change if that's okay!

Reply:
`sourcegraph-service-map` sounds good!